src/nmdc_submission_schema/schema/nmdc_submission_schema_base.yaml

```diff
@@ -259,6 +259,51 @@ slots:
     range: JgiMtInterface
```

Comment:
@pkalita-lbl I am assuming I don't need to include this??

Reply:
range:JgiMtInterface

is that needed here? Or should I remove that range? I assumed it isn't needed, since I assign the slots in the class section.
I guess a better question is what does this metaslot for?

Reply:
It is needed. It's saying that the `jgi_mt_data` slot holds instances of the `JgiMtInterface` class. That's going to be true matter what slots the `JgiMtInterface` class has.

Reply:
for metadata slots in columns... the range is not jgi_mt_data
it's the actual range {text}